docs/walkthroughs/provision-keycloak-apb.adoc

```diff
@@ -25,17 +25,17 @@ This service type can be shared between different Keycloak services.
 After the service has been provisioned, you are then able to open the Keycloak web console
 with the address 'http://keycloak-<project-name>.<routing-suffix>' and the ```<project-name>``` realm should be available.
```

Comment:
can this address be put in back ticks rather?

Reply:
For some reason if the address is within back ticks, it renders with a `;` a the end: ```http://keycloak-<project-name>.<routing-suffix>;```